main/demo_tasks/quickconnect_v2_demo/quickconnect_v2_demo_config.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * ESP32-C3 Featured FreeRTOS IoT Integration V202204.00
+ * Copyright (C) 2022 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * https://www.FreeRTOS.org
+ * https://github.com/FreeRTOS
+ *
+ */
+
+#ifndef QUICKCONNECT_V2_DEMO_CONFIG_H
+#define QUICKCONNECT_V2_DEMO_CONFIG_H
+
+/* ESP-IDF sdkconfig include. */
+#include <sdkconfig.h>
+
+#if CONFIG_GRI_RUN_QUALIFICATION_TEST
+    #include "qualification_wrapper_config.h"
+#endif /* CONFIG_GRI_RUN_QUALIFICATION_TEST */
+
+/* *INDENT-OFF* */
+    #ifdef __cplusplus
+        extern "C" {
+    #endif
+/* *INDENT-ON* */
+
+/**
+ * @brief Size of statically allocated buffers for holding topic names and
+ * payloads.
+ */
+#define quickconnectv2configSTRING_BUFFER_LENGTH                    ( ( unsigned int ) ( CONFIG_GRI_SUB_PUB_UNSUB_DEMO_STRING_BUFFER_LENGTH ) )
```

Comment:
Is this string buffer length enough? The PubSub demo is publishing a much shorter string.